packages/math-input/src/store/pager-reducer.ts

```diff
@@ -41,70 +59,41 @@ const pagerReducer = function (
                 dx: 0,
             };
 
-        case "SetPageSize":
+        case SetPageSizeActionType:
             return {
                 ...state,
-                // @ts-expect-error [FEI-5003] - TS2339 - Property 'pageWidthPx' does not exist on type '{ type: string; }'.
                 pageWidthPx: action.pageWidthPx,
             };
 
-        case "PressKey":
-            // @ts-expect-error [FEI-5003] - TS2339 - Property 'key' does not exist on type '{ type: string; }'.
+        case PressKeyActionType:
             const keyConfig = KeyConfigs[action.key];
 
             // Reset the keypad page if the user performs a math operation.
             if (
                 keyConfig.type === KeyTypes.VALUE ||
                 keyConfig.type === KeyTypes.OPERATOR
             ) {
-                return pagerReducer(state, {type: "ResetKeypadPage"});
+                return {
+                    ...state,
+                    animateToPosition: true,
+                    // We start at the right-most page.
+                    currentPage: getDefaultPage(state.numPages),
+                    dx: 0,
+                };
             }
             return state;
 
-        case "ResetKeypadPage":
-            return {
-                ...state,
-                animateToPosition: true,
-                // We start at the right-most page.
-                currentPage: getDefaultPage(state.numPages),
-                dx: 0,
-            };
-
-        case "PageKeypadRight":
-            const nextPage = Math.min(
-                state.currentPage + 1,
-                state.numPages - 1,
-            );
-            return {
-                ...state,
-                animateToPosition: true,
-                currentPage: nextPage,
-                dx: 0,
-            };
-
-        case "PageKeypadLeft":
-            const prevPage = Math.max(state.currentPage - 1, 0);
-            return {
-                ...state,
-                animateToPosition: true,
-                currentPage: prevPage,
-                dx: 0,
-            };
```

Comment:
Did we lose these action reducer cases now?  Or are they merged in somewhere else that I'm missing? (or unused?)

Reply:
So these cases were (as far as I could tell) only called by one other action each. I felt like it was unnecessarily complex (and more complex to type), so I took the logic and merged it with the cases that needed it.